quickstarts/tidbcloud/config.yml

```diff
@@ -0,0 +1,50 @@
+# Sets the URL name of the quickstart on public I/O (required)
```

Comment:
Seems like you're not referencing the datasource you created. This can be done by putting: 

`
dataSourceIds:
 -id-of-data-source

`

Reply:
[Example: ](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#quickstarts)

Reply:
Got, I add a content "dataSourceIds: tidbcloud"